scripts/docker-entrypoint.sh

```diff
@@ -126,6 +129,25 @@ else
    export SECRET_CASSANDRA_PASSWORD=${CASSANDRA_PASSWORD}
 fi
 
+if [ -e "$hazelcast_username_file" ]; then
+   export SECRET_HZ_CS_AUTH_USERNAME=$(<${hazelcast_username_file})
+else
+   export SECRET_HZ_CS_AUTH_USERNAME=${HZ_CS_AUTH_USERNAME}
+fi
+
+if [ -e "$hazelcast_password_file" ]; then
+   export SECRET_HZ_CS_AUTH_PASSWORD=$(<${hazelcast_password_file})
+else
+   export SECRET_HZ_CS_AUTH_PASSWORD=${HZ_CS_AUTH_PASSWORD}
+fi
+
+if [ "$CLIENT_MODE" = true ]; then
```

Comment:
Will it be `CLIENT_MODE` or `HZ_CLIENT_MODE`?

Reply:
Yes, it should be HZ_CLIENT_MODE.